llama_index/node_parser/extractors/marvin_entity_extractor.py

```diff
@@ -0,0 +1,59 @@
+from typing import List, Optional, Sequence, Type, Dict
+
+from llama_index.schema import BaseNode, TextNode
+
+from llama_index.node_parser.extractors.metadata_extractors import (
+    MetadataFeatureExtractor,
+)
+
+from marvin import AIModel
```

Comment:
off the top of my head we'd probably want this to be a lazy dep (lazy import within `__init__`), similar to our other integrations

Reply:
ok ... will change

Reply:
and yes will add a notebook

Reply:
should I include the notebook in the same directory or under examples somewhere or somewhere else?

Reply:
should be under llama_index/docs/examples/metadata_extraction  - https://github.com/jerryjliu/llama_index/tree/main/docs/examples/metadata_extraction

Reply:
yeah ... I just changed ... also updated to added @classmethod
    def class_name(cls) -> str:
        """Get class name."""
        return "MarvinEntityExtractor"

testing now

Reply:
Should be good to go now.  Moved the notebook and added the classmethod to the extractor.  I think I got everything, but let me know.